cli/cmd/vuln_host.go

```diff
@@ -360,6 +387,18 @@ func init() {
 		vulHostShowAssessmentCmd.Flags(),
 	)
 
+	setFailOnSeverityFlag(
+		vulHostListCvesCmd.Flags(),
+		vulHostShowAssessmentCmd.Flags(),
+		vulHostScanPkgManifestCmd.Flags(),
+	)
+
+	setFailOnFixableFlag(
+		vulHostListCvesCmd.Flags(),
```

Comment:
same 🤔

Reply:
Same as above, it was an idea to add it to list cve's aswell.